File: 代码相关/测试可移动菜单项/测试可移动菜单项/ViewController.m
//
//  ViewController.m
//  测试可移动菜单项
//
//  Created by lin on 14-6-6.
//  Copyright (c) 2014年 北京致远. All rights reserved.
//

#import "ViewController.h"
#import "SyLCustomMenuView.h"


@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
    SyLCustomMenuView *vCustomMenuView =[[SyLCustomMenuView alloc] initWithFrame:CGRectMake(0, 100, 320, 450) withCloum:3 Row:10];
    [self.view addSubview:vCustomMenuView];
    [vCustomMenuView release];
    self.automaticallyAdjustsScrollViewInsets = NO;
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: 代码相关/swift/测试nib创建自定义Cell/测试nib创建自定义Cell/ViewController.swift
//
//  ViewController.swift
//  Cell
//
//  Created by lin on 14-7-10.
//  Copyright (c) 2014 lin. All rights reserved.
//

import UIKit

class ViewController: UIViewController,UITableViewDataSource {
                            
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


    func tableView(tableView: UITableView!, numberOfRowsInSection section: Int) -> Int
    {
        return 4;
    }
    // Row display. Implementers should *always* try to reuse cells by setting each cell's reuseIdentifier and querying for available reusable cells with dequeueReusableCellWithIdentifier:
    // Cell gets various attributes set automatically based on table (separators) and data source (accessory views, editing controls)
    
    func tableView(tableView: UITableView!, cellForRowAtIndexPath indexPath: NSIndexPath!) -> UITableViewCell!{
        let cell:TestCell = tableView.dequeueReusableCellWithIdentifier("testCell") as TestCell
        cell.lable.text = "this is the\(indexPath.row)"
        return cell;
    }
}

